### ISSUE TITLE
Port functionality to n300 card (single device)

### ISSUE THREAD

**Issue by saichandax** [tenstorrent/tt-metal#13406]
Enable support for BERT-Tiny model using TTNN ops and port functionality to n300 card (single device).

**Comment by vigneshkeerthivasanx** [comment]
Draft PR is created for the Bert-Tiny 
Corresponding PR: #13471 
Demo results in n300 is recorded here : [Link](https://docs.google.com/spreadsheets/d/1dBKFjR5tvsdn3oCvyMyOX1D_z_LwMWEI-q7MSQjvo_o/edit?gid=1101027898#gid=1101027898)

**Comment by vigneshkeerthivasanx** [comment]
@mbahnasTT , Bert-Tiny model is completed and the PCC for all sub-modules > 0.99.  But the PCC of the entire model has dropped  after the final linear layer to ~0.96 and ~0.97 after splitting start_logits and end_logits respectively.
The models accuracy for 100 samples are as follows:

Accuracy between TTNN model and SQUADv2 reference: 

```
Eval: exact: 14.423076923076923 
F1: 22.086671341479036
```

Accuracy between PyTorch model and SQUADv2 reference: 

```
exact: 19.23076923076923 
F1: 24.28325947556717
```

Accuracy between TTNN and PyTorch model: 
```
exact: 58.65384615384615
F1: 0.5875457875457875
```

Pytorch results does not align with the dataset for most of the cases.
However, Pytorch and TTNN results match in most cases, and the results can be found here: [Link](https://docs.google.com/spreadsheets/d/1dBKFjR5tvsdn3oCvyMyOX1D_z_LwMWEI-q7MSQjvo_o/edit?gid=1101027898#gid=1101027898)
Corresponding Draft PR : #13471 
Can we proceed with this model with the above results?

**Comment by mbahnasTT** [comment]
@vigneshkeerthivasanx please proceed with this model, and file an issue with these details, and keep track of the issue # i the main model status sheet. Thanks!

**Comment by saichandax** [comment]
Corresponding PR #13471 is merged.
